### ISSUE TITLE
Changing Mix or FTB transition rate on TVS HD is not logical

### ISSUE THREAD

**Issue by Gartom** [bitfocus/companion-module-bmd-atem#122]
When trying to setup a button for changing the transition rate for Mix or FTB, there is a slider available to select a value. The value doesn't correspond to a logical settings in the ATEM mixer. Some examples:

A value of 10 corresponds to 0:10 s
A value of 24 corresponds to 0:24 s
A value of 25 corresponds to 1:00 s
A value of 49 corresponds to 1:24 s
A value of 50 corresponds to 2:00 s

Tested on latest stable release of Companion, i.e. v2.1.2 (2.1.2-f4bcaf3-2679)

**Comment by Julusian** [comment]
The logic is that it is counting in frames. That is the unit that the atem protocol uses, so is simplest for this module to expose the same thing

**Comment by Gartom** [comment]
Yes, I figured that the RATE in ATEM SW control could be defined as related to frames in some way but I still don't get it. We are running our ATEM TVS HD in 1080p50 and  wouldn't a value of 50 then correspond to 1s instead of a value of 25?

I haven't checked what happens if I try to manually enter something else in the ATEM SW control as this was something I just stumbled on when creating a sequence for a Companion button that involved several different modules.
